### PR TITLE
Read data from aux adc for adbms 2950

### LINKED ISSUE #51: [HV Plate] - Support fault diagnostics
### Description

Support fetching diagnostics via multiplexer sequence and functions in hv_plate.c

### Acceptance Criteria

Do it.

### Proposed Solution

Do it.  Use available commands from the driver.  
- VREGUV
- VREGOV
- VDDUV
- VDIGUV
- VDIGOV
- VDE
- VDEL
- OSCFLT
- NOCLK
- SPIFLT
- THSD
- RESET

### LINKED ISSUE #52: [HV Plate] - Support fetching auxillary diagnostics
### Description

Support HV plate diagnostic measurements via aux adc.

### Acceptance Criteria

Do it.  Low frequency reads.

### Proposed Solution

Add to hv_plate.c and sequence as appropriate (ADX, RDXA,B,C):
- VREG
- TMP1
- VREF1P25
- EPAD
- VDIG
- VDD
- OSCCNT
- TMP2
- VDIV

### PR DESCRIPTION
**Pull request by suryatho**
## Changes

Read diagnostic data and flags from adbms2950 chip
send data over can every 5seconds

## Notes

_Any other notes go here_

## Test Cases

- Case A
- Edge case
- ...

## To Do

_Any remaining things that need to get done_

- [ ] item 1
- [ ] ...

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please reach out to your Project Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [ ] No merge conflicts
- [ ] All checks passing
- [ ] Remove any non-applicable sections of this template
- [ ] Assign the PR to yourself
- [ ] Request reviewers & ping on Slack
- [ ] PR is linked to the ticket (fill in the closes line below)

Closes #51
Closes #52

### CODE REVIEW

**Comment by suryatho** [comment]
Should I be using the debug thread at all or should I just send it when I read the data in the hvplate thread?

File: Core/Src/can_messages.c [hunk 1]
```diff
@@ -697,3 +697,49 @@ void send_control_signals(const uint8_t *signals)
...
+	float vdd = hv_plate->vdd * 100;
+	float tmp2 = hv_plate->tmp2 * 100;
+	float vdiv = hv_plate->vdiv * 100;
+
+	bitstream_t bitstream;
+	uint8_t bitstream_data[8];
+
+	// Send first msg
+	bitstream_init(&bitstream, bitstream_data, 8);
+	bitstream_add(&bitstream, *((uint32_t *)&hv_plate->adbms_flags), 12);
```
Comment (suryatho) [inline Core/Src/can_messages.c]: Is this type of reinterpret cast bad? I don't really see any way else to go about it.

**Review by caiodasilva2005 — APPROVED** [review]
lgtm!
